crates/noirc_evaluator/src/ssa/builtin.rs

```diff
@@ -37,34 +46,42 @@ impl Opcode {
         match self {
             Opcode::LowLevel(op) => {
                 match op {
+                    // Pointers do not overflow
                     BlackBoxFunc::SHA256
                     | BlackBoxFunc::Blake2s
                     | BlackBoxFunc::Pedersen
-                    | BlackBoxFunc::FixedBaseScalarMul => BigUint::zero(), //pointers do not overflow
+                    | BlackBoxFunc::FixedBaseScalarMul => BigUint::zero(),
+                    // Verify returns zero or one
                     BlackBoxFunc::SchnorrVerify
                     | BlackBoxFunc::EcdsaSecp256k1
-                    | BlackBoxFunc::MerkleMembership => BigUint::one(), //verify returns 0 or 1
+                    | BlackBoxFunc::MerkleMembership => BigUint::one(),
                     BlackBoxFunc::HashToField128Security => ObjectType::NativeField.max_size(),
-                    _ => todo!("max value must be implemented for opcode {} ", op),
+                    BlackBoxFunc::RANGE
+                    | BlackBoxFunc::AES
+                    | BlackBoxFunc::AND
```

Comment:
Note; The benefit of removing the wildcard reveals that we are panicking on the Range, And, and Xor case

Reply:
This PR is blocked by this comment.

Reply:
You can use an `unreachable("unexpected opcode")` here if you want.

Reply:
Why is range, and and  xor unexpected?